packages/button/package.json

```diff
@@ -5,11 +5,13 @@
   "license": "Apache-2.0",
   "repository": "pluralsight/design-system",
   "author": "jaketrent",
-  "main": "index.js",
+  "main": "react.js",
```

Comment:
This definitely makes this a breaking change, but I don't think your commit specifies that yet.  Please follow this spec for that:

https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary